core/src/test/java/com/radixdlt/p2p/test/P2PTestNetworkRunner.java

```diff
@@ -188,6 +194,48 @@ private static Injector createInjector(
       throws ParseException {
     final var properties = RuntimeProperties.fromCommandLineArgs(new String[] {});
     return Guice.createInjector(
+        new AbstractModule() {
+
+          // FIXME: Replacing implementation with the RocksDB version does not work out of the box
+          // because NodeRustEnvironment can't
+          // be properly instantiated (error on Rust side).
+          @Override
+          protected void configure() {
+            bind(AddressBookPersistence.class).to(BerkeleyAddressBookStore.class);
+            bind(PersistentSafetyStateStore.class).to(BerkeleySafetyStateStore.class);
+            Multibinder.newSetBinder(binder(), NodeAutoCloseable.class)
+                .addBinding()
+                .to(BerkeleySafetyStateStore.class);
+          }
+
+          @Provides
+          @Singleton
+          BerkeleySafetyStateStore safetyStateStore(
```

Comment:
Minor - do we need a `BerkeleySafetyStateStore` here?

Reply:
Reworked module.